exonum-java-binding/core/src/main/java/com/exonum/binding/core/transaction/TransactionMethod.java

```diff
@@ -31,9 +31,9 @@
  *   <li>be public
  *   <li>have exactly two parameters - the
  *       {@linkplain TransactionMessage#getPayload() serialized transaction arguments} of type
```

Comment:
just the 'transaction arguments', either as `byte[]` or as a protobuf message. Please also add that the protobuf messages will be deserialized using a `public static {@code #parseFrom(byte[])} method`